Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -203,14 +204,9 @@ @interface SettingsViewController () <UITextFieldDelegate, MXKCountryPickerViewC
 SettingsDiscoveryTableViewSectionDelegate, SettingsDiscoveryViewModelCoordinatorDelegate,
 SettingsIdentityServerCoordinatorBridgePresenterDelegate,
 ServiceTermsModalCoordinatorBridgePresenterDelegate,
-<<<<<<< HEAD
 TableViewSectionsDelegate/*,
-ThreadsBetaCoordinatorBridgePresenterDelegate*/>
-=======
-TableViewSectionsDelegate,
 ThreadsBetaCoordinatorBridgePresenterDelegate,
-ChangePasswordCoordinatorBridgePresenterDelegate>
->>>>>>> v1.8.20
+ChangePasswordCoordinatorBridgePresenterDelegate*/>
```

Comment:
Could we use this new delegate ?

Reply:
I don't think we could use it at this time. Probably when we realign the change password feature onto Element.

Reply:
Forget my previous comment. The delegate is needed for the changepassword feature in Tchap too.

Reply:
It has been changed in last commit.